tests/std/tests/P0355R7_calendars_and_time_zones_io/duration_output_tests.h

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft Corporation.

```

Comment:
I was thinking this would be a convenient place to group the [time.parse], [time.format], and [time.duration.io] tests, given the preference for a smaller number of larger tests, while also not lumping all of it into a single large cpp. I'm happy to move it all to test.cpp if that's preferred.

Reply:
When separate test directories want to share common machinery, we add a header to https://github.com/microsoft/STL/tree/master/tests/std/include (with an `.hpp` extension). I think it would be reasonable to add a header there now, even if it's used by only one test directory to start with - then we would follow the pattern that @mnatsuhara noted (where test directories don't have local header files, except in special situations), and we will be prepared for additional tests in the future as @MattStephanson noted, without needing to move code from a `test.cpp` into a header at that time. (git and GitHub make it easy to view file moves, but harder to notice when part of a file's contents have moved exactly.)

Reply:
The header was supposed to organize tests of different features, rather than common code, so I've moved everything into `test.cpp`.